exporters/tests/test_pelorus_webhook_handler.py

```diff
@@ -244,3 +247,162 @@ async def test_pelorus_receive_pelorus_payload_error(headers, json_payload):
         await handler._receive_pelorus_payload(json_payload_data)
     assert http_exception.value.detail == "Invalid payload."
     assert http_exception.value.status_code == http.HTTPStatus.UNPROCESSABLE_ENTITY
+
+
+# fmt: off
```

Comment:
what are these comments?

Reply:
From the black documentation:
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html

"Black reformats entire files in place. It doesn’t reformat  [...]  blocks that start with # fmt: off and end with # fmt: on.